srcpkgs/nvme-cli/template

```diff
@@ -1,28 +1,19 @@
 # Template file for 'nvme-cli'
 pkgname=nvme-cli
-version=1.13
-revision=2
-build_style=gnu-makefile
-# yeah, it must be 0 to enable
-make_build_args="LIBUUID=0"
-make_install_args="SBINDIR=/usr/bin UDEVDIR=/usr/lib/udev"
-make_install_target="install-spec"
-makedepends="libuuid-devel"
+version=2.0
+revision=1
+build_style=meson
+configure_args="-Ddocs=man"
+hostmakedepends="pkg-config"
+makedepends="libnvme-devel zlib-devel json-c-devel libuuid-devel libhugetlbfs-devel"
 short_desc="NVMe management command line interface"
-maintainer="Peter Froehlich <peter.hans.froehlich@gmail.com>"
-license="GPL-2.0-only"
+maintainer="Subhaditya Nath <sn03.general@gmail.com>"
+license="LGPL-2.0-or-later"
```

Comment:
This doesn't match the LICENSE in the repo.

Reply:
https://github.com/linux-nvme/nvme-cli/blob/5a36baba322e7979175ed9dfeede0a96e29efc37/meson.build#L5

Reply:
https://github.com/linux-nvme/nvme-cli/blob/5a36baba322e7979175ed9dfeede0a96e29efc37/LICENSE

Reply:
See https://github.com/linux-nvme/nvme-cli/issues/1508, revert this line,